app/components/game-card.js

```diff
@@ -12,9 +12,7 @@ export default Ember.Component.extend({
 
   attributeBindings: ['draggable'],
 
-  classNames: ['game-card'],
-
-  classNameBindings: ['isTapped:tapped', 'readOnly::cursor-move'],
+  classNameBindings: ['readOnly::cursor-move'],
```

Comment:
Yeah, I think I had to put it one level down in the markup so that the magnification div wasn't a child of the tapped div.